add udf to build and list (#8062)
fishtown-analytics/docs.getdbt.com: website/docs/reference/commands/build.md

---

The `dbt build` command will:
- run [models](/docs/build/models)

> - test [tests](/docs/build/data-tests)

Reviewer: Suggesting to change the verbs used here, please let me know if they're correct:
```suggestion
- run [tests](/docs/build/data-tests)
```

Author: i love this and it totally makes sense to change the verbs! but i also like the 'test tests' because we're essentially testing tests and using the 'dbt test' command, not 'dbt run'. do you feel strongly that it should be 'run'? i can maybe add a 'test [tests] using `dbt test` command..' or something like that for the rest of them?

Reviewer: `test tests` sounds good to me! 👍

> - snapshot [snapshots](/docs/build/snapshots)

Reviewer: ```suggestion
- execute [snapshots](/docs/build/snapshots)
```

Author: hey @luna-bianca - for snapshot and seeds - its teh same feedback as above since the command is snapshot/seed.

> - seed [seeds](/docs/build/seeds)

Reviewer: ```suggestion
- load [seeds](/docs/build/seeds)
```

> - run [user-defined functions](/docs/build/udfs) (available from dbt Core v1.11 or Fusion)

Reviewer: ```suggestion
- run [user-defined functions](/docs/build/udfs) (available from dbt Core v1.11 and in the <Constant name="fusion_engine")
```

> - run [user-defined functions](/docs/build/udfs) (available from dbt Core v1.11 and in the <Constant name="fusion_engine")

Author: ```suggestion
- run [user-defined functions](/docs/build/udfs) (available from dbt Core v1.11 and in the <Constant name="fusion_engine" />)
```

> - run [user-defined functions](/docs/build/udfs) (available from dbt Core v1.11 and in the <Constant name="fusion_engine" />)

Reviewer 2: can this say "build" user-defined functions instead of "run"? that's confusing given that "run" is just for models

Done. PASS=7 WARN=0 ERROR=0 SKIP=0 TOTAL=7
```

## Functions

> _Available from dbt Core v1.11 or Fusion_

Author: ```suggestion
_Available from dbt Core v1.11 and in the <Constant name="fusion_engine" />_
```

> The `build` command runs [user-defined functions](/docs/build/udfs) as part of the DAG execution. To build or rebuild only `functions` in your project, run `dbt build --select "resource_type:function"`. For example:

Reviewer 2: again, can we say the build command "builds" user-defined functions - run is just for models

```bash
dbt build --select "resource_type:function"
dbt-fusion 2.0.0-preview.45

>  Succeeded [  0.98s] function dbt_mwong.whoami (function)

Reviewer: Suggesting to use a generic `dbt_schema` in the sample:
```suggestion
 Succeeded [  0.98s] function dbt_schema.whoami (function)
```

Author: fair point!

>  Succeeded [  1.12s] function dbt_mwong.area_of_circle (function)

Reviewer: ```suggestion
 Succeeded [  1.12s] function dbt_schema.area_of_circle (function)
```
